web/src/routes/OffersPage.tsx

```diff
@@ -39,15 +38,6 @@ const OffersPage = ({ city, cityCode, languageCode }: CityRouteProps): ReactElem
           // the url stored in the sprungbrett offer is the url of the endpoint
           path = pathnameFromRouteInformation({ route: SPRUNGBRETT_OFFER_ROUTE, cityCode, languageCode })
         }
-        if (offer.alias === MALTE_HELP_FORM_OFFER_ROUTE) {
-          path = pathnameFromRouteInformation({ route: MALTE_HELP_FORM_OFFER_ROUTE, cityCode, languageCode })
-          // ignore zammad url (postData) for the tile model
-          return new TileModel({
-            title: t(offer.title),
-            path,
-            thumbnail: offer.thumbnail,
-          })
-        }
```

Comment:
I think this should be removed when we actually remove the offers tab. Otherwise the helpform appears in the offers tab but cannot be clicked on.

Reply:
I will just filter it out, good point :+1: